src/main/java/network/brightspots/rcv/OutputWriter.java

```diff
@@ -696,17 +697,19 @@ String writeRcTabCvrCsv(
       CSVFormat format = CSVFormat.DEFAULT.builder().setNullString("").build();
       csvPrinter = new CSVPrinter(writer, format);
       // print header:
-      // ContestId, TabulatorId, BatchId, RecordId, Precinct, Precinct Portion, rank 1 selection,
-      // rank 2 selection, ... rank maxRanks selection
-      csvPrinter.print("Source Filepath");
-      csvPrinter.print("CVR Provider");
-      csvPrinter.print("Contest Id");
-      csvPrinter.print("Tabulator Id");
-      csvPrinter.print("Batch Id");
-      csvPrinter.print("Record Id");
+      // RCTab CVR Id, ContestId, TabulatorId, BatchId, RecordId, Precinct, Precinct Portion,
+      csvPrinter.print("Source Filepath"); // CvrSource.getFilePath()
+      csvPrinter.print("CVR Provider"); // CvrSource.getProvider()
+      csvPrinter.print("Contest Id"); // CastVoteRecord.getContestId()
+      csvPrinter.print("RCTab CVR Id"); // CastVoteRecord.getId()
+      csvPrinter.print("Tabulator Id"); // CastVoteRecord.getTabulatorId
+      csvPrinter.print("Batch Id"); // CastVoteRecord.getSlice(TabulateBySlice.BATCH)
+      csvPrinter.print("Vendor Id"); // CastVoteRecord.getSuppliedId
+      // CastVoteRecord.getSlice(ContestConfig.TabulateBySlice.PRECINCT)
       csvPrinter.print("Precinct");
```

Comment:
Probably can make this line match the rest on one line (and therefore make it more readable) by using
`csvPrinter.print("Precinct"); // CastVoteRecord.getSlice(TabulateBySlice.PRECINCT)`, or if not, just `(PRECINCT)`